### PR TITLE
Updated german translation (951 - 955)

### PR DESCRIPTION
**Pull request by Neui**
Update the german translations for 951 to 954.

### CODE REVIEW

File: lang/de_DE.trn [hunk 1]
```diff
@@ -2044,10 +2044,10 @@ You may not be able to access all functions at this resolution.
 t950 MCEdit unterstützt keine Fensterauflösungen unter 1000x700.
 Du wirst unter dieser Auflösung vielleicht nicht auf alle Funktionen zugreifen können.
 o951 Found a previous portable installation
-t951 
+t951 Eine vorherige portabel Installation wurde gefunden
```
Comment (robske110) [inline lang/de_DE.trn]: change portabel zu portable !!!

**Comment by robske110** [comment]
Look at my note

File: lang/de_DE.trn (hunk 1 above)
Comment (Neui) [inline lang/de_DE.trn]: `portable` is english, `portabel` is (old) german. We already use `portabel`, so for consistency, i also used that.

Comment (robske110) [inline lang/de_DE.trn]: Aber hier macht portabel kein sinn
Es ist eine PORTABLE installation
Nicht eine portabel installation

Comment (robske110) [inline lang/de_DE.trn]: Und bei den beiden linien ist es logisch, bei denen ist keine/es macht grammtokalish sinn

**Comment by Neui** [comment]
Meh i fucked up some commits trying to solve a merge confilct. Wait till i "removed" the uneeded ones

File: lang/de_DE.trn [hunk 2]
```diff
@@ -2044,10 +2044,10 @@ You may not be able to access all functions at this resolution.
 t950 MCEdit unterstützt keine Fensterauflösungen unter 1000x700.
 Du wirst unter dieser Auflösung vielleicht nicht auf alle Funktionen zugreifen können.
 o951 Found a previous portable installation
-t951 
+t951 Eine vorherige transportable Installation wurde gefunden
```
Comment (robske110) [inline lang/de_DE.trn]: 👍

**Comment by Khroki** [comment]
This request has conflicts, could you resubmit this request?

**Comment by Neui** [comment]
With the help in IRC (by jhass), i now added the translation for 955. (And thus no more conflicts)

**Pull request merged**